polaris/evaluate/_metric.py

```diff
@@ -30,6 +31,28 @@ def spearman(y_true: np.ndarray, y_pred: np.ndarray):
     return stats.spearmanr(y_true, y_pred).statistic
 
 
+def absolute_average_fold_error(y_true: np.ndarray, y_pred: np.ndarray):
+    """
+    Calculate the Absolute Average Fold Error (AAFE) metric.
+
+    Parameters:
+    y_true : array-like of shape (n_samples,)
+        The true target values.
+    y_pred : array-like of shape (n_samples,)
+        The predicted target values.
+
+    Returns:
+    aafe : float
+        The Absolute Average Fold Error.
```

Comment:
Should be consistent with the docstring style in the rest of the repo, so `Args` instead of `Parameters` and no type annotations in the docstring.